spec/models/sms/sms_decoder_spec.rb

```diff
@@ -557,9 +633,18 @@ def assert_decoding(options)
     @response = Sms::Decoder.new(@msg).decode
     @response.save!
 
+    @response
+  end
+
+  # tests that a decoding was successful
+  def assert_decoding(options)
```

Comment:
Can you do me a favor and change `assert_decoding` to `expect_decoding` and same for `assert_decoding_fail`? `assert` is old skool :)